app/assets/stylesheets/modules/applicationfooter.scss

```diff
@@ -1,11 +1,18 @@
 @import "variables";
 
+body {
+  min-height: 100%;
+  position: relative;
+}
+
 .application-footer {
   background: $blue;
   bottom: 0;
   color: $white;
-  height: 100px;
+  height: 150px;
   left: 0;
+  right: 0;
```

Comment:
Properties should be ordered background, bottom, color, height, left, line-height, padding, position, right, text-align, width